blog/snippets/playwright/basic-get-image.js

```diff
@@ -0,0 +1,21 @@
+const { chromium } = require("playwright");
+const axios = require("axios");
+const fs = require("fs");
+
+(async () => {
+  const browser = await chromium.launch();
+  const page = await browser.newPage();
+  await page.goto("https://danube-webshop.herokuapp.com");
+  const url = await page.$eval("img", (el) => el.src);
+
+  axios
```

Comment:
We can use `await` syntax here to make it shorter. Also explain the use of `axios` in the main text.

Reply:
sorry, my comment was terrible. The `axios` calls can also use `await`

Reply:
Understood, makes total sense. Applying.